faq/index.html

```diff
@@ -3745,15 +3745,15 @@ <h2 id="artificial-intelligence-and-cc-licenses">Artificial intelligence and CC
 title="Considerations for licensors and licensees - Creative Commons Wiki">reusers
 of works</a> before they do so in order to avoid inadvertent violations.</p>
 <h2 id="notes">Notes</h2>
-<aside id="footnotes" class="footnotes footnotes-end-of-document"
+<section id="footnotes" class="footnotes footnotes-end-of-document"
```

Comment:
This isn't semantically correct. The `h2` of "notes" would need to be inside the `section` element, as an `section` generally requires a heading element to denote it.

It could be argued that of the two, the `aside` is actually a more appropriate element, but it too would need the `h2` to be within it as well. An `aside does not require a heading element within it, but if one appears for it, it must appear within it.


The MOST semantic, and likely correct route, would be for the endnotes to appear (with the associated `h2`) within a `footer` element (instead of a `section` or `aside`). As they are meant to be footnotes for the `main` section, that is the next parent level sectioning element and they would then be semantically associated as the "footer of the main area", which is much more accurate to what the footnotes are in this case.

sooo...

`````
<footer>
<h2 id="notes">Notes</h2>
notes here
</footer>
</main>
`````

As `divs` are semantically meaningless whether it appears before or after the closing `div` tag is semantically irrelevant, just whichever one wouldn't throw off the visual styling.